validation/src/main/scala/hmda/validation/SubmissionLarStats.scala

```diff
@@ -40,15 +43,26 @@ class SubmissionLarStats(submissionId: SubmissionId) extends HmdaPersistentActor
   }
 
   override def receiveCommand: Receive = super.receiveCommand orElse {
+    case s: String =>
+      totalSubmittedLars = totalSubmittedLars + 1
+
     case LarValidated(_, _) =>
-      totalLars = totalLars + 1
+      totalValidatedLars = totalValidatedLars + 1
+
+    case CountSubmittedLarsInSubmission =>
```

Comment:
Oh, I think I figured it out. Is it because Q130 is a Quality edit (as opposed to Macro), and the "validated" count doesn't get calculated soon enough?